docs/Developers/Newcomers_Guide.rst

```diff
@@ -252,16 +252,26 @@ If you know you have more work to do on this ``Pull Request`` before it is
 ready to be accepted, you can optionally indicate this to other developers
 by starting your ``Pull Request`` title with ``wip:`` (case-insensitive).
 
-Step 7. Waiting for Review
+Step 7. Reviewing Process
 --------------------------
 
-After creating a Pull Request, your PR is open to the review process (to read
-more about it, have patience and it is explained on the next step), and all
-you can do is wait. The best thing you can do while at this step is review
-other people's PRs. Not only will this help the maintainers with the workload,
-but this is one of the three core steps towards becoming a full-norm coalaian.
+After creating a Pull Request, your PR is open to the review process.
 
-For more information about reviewing code, check out this `link <http://coala.io/reviewing>`_.
+You may ask for a review after you have done a review as well.
+When you ask for it, please provide a link of something you just reviewed.
+We need to do this so everyone gets timely reviews and people who are
+helping with this get preferred.
+Go to `our list of PRs that are awaiting reviews <http://coala.io/review>`_
+and choose a PR that is labeled with `difficulty/newcomer`. These PRs are from
+other newcomers and should require little understanding of the coala codebase
+to help you learning the reviewing process. If there are no newcomer PRs
+waiting for reviews you can still take a look at any of the listed PRs.
+
+Not only will this help the maintainers with the workload,
+but this is one of the three core steps towards becoming a full-grown coalian.
```

Comment:
coalian? Is that official? I thought it's coalaian, ugh... nvm... sounds like co-alien though :3